src/main/java/me/mrCookieSlime/Slimefun/Events/DisenchanterEvent.java

```diff
@@ -0,0 +1,42 @@
+package me.mrCookieSlime.Slimefun.Events;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.bukkit.inventory.ItemStack;
+
+public class DisenchanterEvent extends Event implements Cancellable {
+	
+	private static final HandlerList handlers = new HandlerList();
+
+	private ItemStack sfItem;
+	private boolean cancelled;
+
+	public HandlerList getHandlers() {
+	    return handlers;
+	}
+	
+	public static HandlerList getHandlerList() {
+        return handlers;
+    }
+	
+	public DisenchanterEvent(ItemStack sfitem){
```

Comment:
Constructor goes before normal methods

Reply:
^ What he said.

Reply:
I'm sorry, but I don't know what you mean. I'm not a native English speaker, I've never used this literally, I've always created events like this. Can you tell me, without any judgment, what should I change about this?

Reply:
Literally move this above the other methods.

Reply:
Okay, thanks